src/main/java/io/cdap/plugin/db/batch/action/vertica/export/VerticaBulkExportAction.java

```diff
@@ -160,5 +176,35 @@ public VerticaConfig(String connectionString, String user, String password, Stri
       this.delimiter = delimiter;
       this.path = path;
     }
+
+    public void validate(FailureCollector failureCollector) {
+      if (Strings.isNullOrEmpty(path)) {
+        failureCollector.addFailure("File path is required.",null)
+          .withConfigProperty(PATH);
+      }
+      if (Strings.isNullOrEmpty(connectionString)) {
+        failureCollector.addFailure("Connection string is required.", null)
+          .withConfigProperty(CONNECTION_STRING);
+      } else if (!connectionString.startsWith(CONNECTION_STRING_PREFIX)) {
+        failureCollector.addFailure(
+          "Connection string has incorrect format.",
+          "Connection string must comply next format - jdbc:vertica://<VerticaHost>:<portNumber>/<databaseName>")
+          .withConfigProperty(CONNECTION_STRING);
+      }
+      if (user == null && password != null) {
+        failureCollector.addFailure(
+          String.format("'%s' is null.", USER),
+          String.format("Please provide both user name and password if database requires " +
+                          "authentication. If not, please remove '%s' and retry.", USER))
+          .withConfigProperty(USER);
+      }
+      if (user != null && password == null) {
+        failureCollector.addFailure(
+          String.format("'%s' is null.", PASSWORD),
```

Comment:
1. use property name from UI widget, not from constant (in this case just start from uppercase letter)
2. check for empty string (Strings class utility methods)
3. "Password is not specified"